crates/starknet-devnet/tests/test_abort_blocks.rs

```diff
@@ -58,23 +59,30 @@ mod abort_blocks_tests {
                 .await
                 .expect("Could not start Devnet");
 
-        let first_block_hash = devnet.create_block().await.unwrap();
-        let second_block_hash = devnet.create_block().await.unwrap();
+        let genesis_block =
+            devnet.json_rpc_client.get_block_with_tx_hashes(BlockId::Tag(BlockTag::Latest)).await;
+        let genesis_block_hash = match genesis_block {
+            Ok(MaybePendingBlockWithTxHashes::Block(b)) => b.block_hash,
+            _ => panic!("Unexpected resp: {genesis_block:?}"),
+        };
```

Comment:
Can be replaced with `devnet.get_latest_block_with_tx_hashes()`

Reply:
Fixed in eedd1e54d19f19b1be1b6cda8cf33be0a0baceb3